.github/workflows/unit-tests.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [windows-latest, ubuntu-latest, macos-latest]
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10, 3.11]
```

Comment:
At least 3.10 needs to be in quotes, so it isn't 3.1

Reply:
Done!